librosa/feature/spectral.py

```diff
@@ -28,7 +28,8 @@
            'mfcc',
            # Deprecated functions
            'logfsgram',
-           'chromagram']
+           'chromagram',
+           'tonnetz']
 
```

Comment:
`tonnetz` should also be added to the feature submodule's docstring index [here](https://github.com/bmcfee/librosa/blob/develop/librosa/feature/__init__.py#L19)